scipy/stats/_multivariate.py

```diff
@@ -760,6 +760,36 @@ def rvs(self, mean=None, cov=1, size=1, random_state=None):
             out = mean + cov_object.colorize(x)
         return out
 
+    def fit(self,data):
+        r"""Fits a multivariante disterbution to data.
+
+        Parameters
+        ----------
+        data : ndarray
+            The data for which the disterbution is fitted to.
+
+        Returns
+        -------
+        h : multivariate_normal
+            Multivariate normal distribution fitted for data
+
+        Notes
+        -----
+        This function uses MLE which probably 
+        isn't the best way to fit multivariante normal disterbution
+
+        References
+        ----------
+        https://numpy.org/doc/stable/reference/generated/numpy.cov.html?highlight=cov#numpy.cov
+
+        .. versionadded:: 1.9.4
```

Comment:
Not in a position to review this but this won't go in until 1.10.0 at the earliest.
```suggestion
        .. versionadded:: 1.10.0
```